index.html

```diff
@@ -4,9 +4,69 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Valera Mykytenko</title>
+    <title>Valera Mykytenko Tables</title>
 </head>
 <body>
+    <table border="2">
+        <thead>
+            <tr>
+                <th>#</th>
+                <th>Назва</th>
+                <th>Репозиторій</th>
+                <th>Github page</th>
+            </tr>
+        </thead>
+        <tbody>
+            <tr>
+                <td>1</td>
+                <td>Перший сайт</td>
+                <td>Відсутній</td>
+                <td>Відсутній</td>
+            </tr>
+            <tr>
+                <td>2</td>
+                <td>Про Git</td>
+                <td>Відсутній</td>
+                <td>Відсутній</td>
+            </tr>
+            <tr>
+                <td>3</td>
+                <td>Git flow</td>
+                <td><a href="https://github.com/kasper93721/git-flow-homework" target="_blank">Git flow</a></td>
+                <td><a href="https://kasper93721.github.io/git-flow-homework/" target="_blank">Link </a></td>
+            </tr>
+            <tr>
+                <td>4</td>
+                <td>Найлегша домашка по Git</td>
+                <td>Відсутній</td>
+                <td>Відсутній</td>
+            </tr>
+            <tr>
+                <td>5</td>
+                <td>Знайомство з HTML + CV</td>
+                <td><a href="https://github.com/kasper93721/CV" target="_blank">CV</a></td>
+                <td><a href="https://kasper93721.github.io/CV/" target="_blank"> Link </a></td>
+            </tr>
+            <tr>
+                <td>6</td>
+                <td>HTML</td>
+                <td>Відсутній</td>
+                <td>Відсутній</td>
+            </tr>
+            <tr>
+                <td>7</td>
+                <td>HTML Form</td>
+                <td><a href="https://github.com/kasper93721/form" target="_blank">form </form></a></td>
```

Comment:
```suggestion
                <td><a href="https://github.com/kasper93721/form" target="_blank">form</a></td>
```
в тебе тут зайвий тег закриття `</form>`! Якщо ти хотів відобразити сам текст, а не тег, то потрібно використовувати для цього спеціальні символи 
https://sabe.io/blog/display-html-tags-plain-text
```txt
<: &lt;
>: &gt;
&: &amp;
": &quot;
```